ddl-scripts/create_tables_postgres.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE IF NOT EXISTS event_journal(
+CREATE TABLE IF NOT EXISTS test_journal(
```

Comment:
Probably not intentional change, or else index needs update as well?

Reply:
thanks, yes, leftover from my testing